e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v1_test.js

```diff
@@ -39,7 +39,7 @@ Scenario('SetAside Default Judgment after judgment error - Spec claim 1v1 - Case
   }
 });
 
-Scenario('Record Judgment Spec claim 1v1 with mark paid in full', async ({I, api_spec}) => {
+Scenario.skip('Record Judgment Spec claim 1v1 with mark paid in full', async ({I, api_spec}) => {
```

Comment:
Is Mark paid in full event is being tested on any other judgment in test? If not then please add mark paid in full event in dJ/JBA tests related to JO.